packages/api/src/brevo-module.ts

```diff
@@ -1,11 +1,16 @@
 import { DynamicModule, Global, Module } from "@nestjs/common";
 
+import { BrevoContactModule } from "./brevo-contact/brevo-contact.module";
+import { BrevoModuleConfig } from "./config/brevo-module.config";
+import { ConfigModule } from "./config/config.module";
+
 @Global()
 @Module({})
 export class BrevoModule {
-    static register(): DynamicModule {
+    static register(config: BrevoModuleConfig): DynamicModule {
         return {
             module: BrevoModule,
+            imports: [BrevoModule, BrevoContactModule, ConfigModule.forRoot(config)],
```

Comment:
The BrevoModule can't import itself?

Reply:
The package contained 2 BrevoModules due to the previous drop of the `comet` prefix of the root module. I renamed one of the modules to 'BrevoApiModule` which contains all the logic for the communication with brevo. Changed in 4d9b89ebf57c92cdc8217a5bb77b5a4b03419635